skycultures/bali/description.md

```diff
@@ -0,0 +1,287 @@
+        <td><img width="255" src="illustrations/descr_pics/Su_U.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Bubu Bolong</notr></td>
+        <td><notr>ᬩᬸᬩᬸᬩᭀᬮᭀᬂ</notr></td>
+        <td>leaky fish trap</td>
+        <td><img width="255" src="illustrations/descr_pics/Su_Pa.jpg"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Prahu Pěgat</notr></td>
+        <td><notr>ᬧ᭄ᬭᬳᬸᬧᭂᬕᬢ᭄</notr></td>
+        <td>a broken ship</td>
+        <td><img width="255" src="illustrations/descr_pics/Su_Po.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Magělut</notr></td>
+        <td><notr>ᬫᬕᭂᬮᬸᬢ᭄</notr></td>
+        <td>quarrel over debt</td>
+        <td><img width="255" src="illustrations/descr_pics/Su_Wa.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Udang</notr></td>
+        <td><notr>ᬳᬸᬤᬂ</notr></td>
+        <td>a prawn <br/>A shrimp.</td>
+        <td><img width="255" src="illustrations/descr_pics/Su_Kli.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Děpat</notr></td>
+        <td><notr>ᬤᭂᬧᬢ᭄</notr></td>
+        <td>demon without body<br/>This is the head of Laweyan. This lintang is the lunar descending node.</td>
+        <td><img width="255" src="illustrations/descr_pics/Sa_U.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Ru</notr></td>
+        <td><notr>ᬭᬸ</notr></td>
+        <td>arrow<br/>Basically bow and arrow </td>
+        <td><img width="255" src="illustrations/descr_pics/Sa_Pa.jpg"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Sungenge</notr></td>
+        <td><notr>ᬲᬸᬗᬾᬗᬾ</notr></td>
+        <td>the Sun</td>
+        <td><img width="255" src="illustrations/descr_pics/Sa_Po.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Puwuh Atarung</notr></td>
+        <td><notr>ᬧᬸᬯᬸᬄᬳᬢᬭᬸᬂ</notr></td>
+        <td>battling quails</td>
+        <td><img width="255" src="illustrations/descr_pics/Sa_Wa.png"/></td>
+    </tr>
+    <tr valign="top">
+        <td><notr>Larung Paglangan</notr></td>
+        <td><notr>ᬮᬭᬸᬂᬧᬕ᭄ᬮᬗᬦ᭄</notr></td>
+        <td>throwing ashes<br/>Cremation will produce the ashes. Here the family needs to throw the ashes to the sea. </td>
+        <td><img width="255" src="illustrations/descr_pics/Sa_Kli.png"/></td>
+    </tr>
+</table>
+</p>
+
+
+
+### Source(s)
+
+The stick figures are interpretations by Youla A Azkarrula based on her pre-doctoral research under supervision of Dr. Dr. Susanne M Hoffmann.
+
+
+## References
+
+ - [#1]: Maass, Alfred, “Astrologische Kalender der Balinesen,” in Koninklijk Bataviaasch Genootschap van Kunsten en Wetenschappen, Feestbundel bij gelegenheid van zijn 150 jarig bestaan 1778-1928, 2 vols. (Weltevreden, 1929), Vol. 2, 126-157.
+ - [#2]: Pengkaji Museum Bali, "Kajian Palelintangan", (Denpasar, 2019).
+ - [#3]: Museum Bali, https://museumbali.org/
+ - [#4]: Kerta Gosa Museum , https://e-museum.klungkungkab.go.id/kerta-gosa/
+
+### Fair Use
```

Comment:
The _References_ section shouldn't contain anything except references. I don't know what _Fair Use_ is supposed to mean (since it just states the obvious), but the Acknowledgements belong in the _Authors_ section, not _References_.

Reply:
Yes, I had made the "Fair Use" statements before we decided to force all SCs into the CC-license system; it's in that way an artifact (that was copied to the new SC) - but I think, it doesn't hurt